components/typography/stories/typography.stories.js

```diff
@@ -18,7 +18,7 @@ import {
 import { TypographyGroup } from "./typography.test.js";
 
 /**
- * Spectrum typography is broken out into several separate components: heading, body, detail, and code. Internationalized typography examples are also shown.
+ * Spectrum typography is broken out into several separate components: [heading](#heading), [body](#body), [detail](#detail), and [code](#code). [Internationalized typography examples](#internationalized-typography) are also shown.
```

Comment:
Typography is the only one of these components where I have pretty strong feelings that we _should_ keep the MDX file. To me, the organization of the MDX file better suits these docs, as opposed to just alphabetically listing everything. This works, but the current flow of the information regarding the typography component seems more accessible and understandable to a user.

Thoughts on reverting the typography changes and keeping the MDX file?